environment.yml

```diff
@@ -11,3 +11,4 @@ dependencies:
     - nextstrain-sphinx-theme>=2020.2
     - sphinx-markdown-tables
     - sphinx-argparse
+    - sphinx-tabs
```

Comment:
Tabs extension was added in an early commit but then its usage was removed in a subsequent commit. I believe it's no longer necessary?